pkg/build/api/types.go

```diff
@@ -8,7 +8,10 @@ import (
 )
 
 // BuildLabel is the key of a Pod label whose value is the Name of a Build which is run.
-const BuildLabel = "build"
+const (
+	BuildAnnotation = "build"
```

Comment:
we have a mix of namespaced and non-namespaced annotations today... not sure if this should be "openshift.io/build". @smarterclayton, is there a word from on high on annotation key?